assets/css/style.scss

```diff
@@ -46,38 +46,56 @@
 		margin-right: 0.5em;
 	}
 }
-.wc-block-grid__product-add-to-cart {
+.wc-block-grid__product-add-to-cart.wp-block-button {
```

Comment:
Is `.wp-block-button` added for specificity? If that's the case, could we add a comment specifying that so it's not removed in the future?